Library/Library/AddBook.aspx

```diff
@@ -0,0 +1,37 @@
+﻿<%@ Page Title="" Language="C#" MasterPageFile="~/Site.Master" AutoEventWireup="true" CodeBehind="AddBook.aspx.cs" Inherits="Library.AddBook" %>
+
+<asp:Content ID="Content3" ContentPlaceHolderID="main" runat="server">
+   
+    <div>
+         <asp:DropDownList ID="AuthorListChose" runat="server" AppendDataBoundItems ="True" AutoPostBack="True" DataSourceID="AuthorListDataSource" DataTextField="FirstName" DataValueField="Id">
```

Comment:
I'd remove the `AutoPostBack` property from the `DropDownList` control. Unless you need to respond to when the user selects an author from the list it's best to avoid the round trip to the server.